community/go-md2man/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Jake Buchholz <tomalok@gmail.com>
+# Maintainer: Jake Buchholz <tomalok@gmail.com>
+
+pkgname=go-md2man
+pkgver=1.0.8
+pkgrel=0
+pkgdesc="Utility to convert markdown to man pages"
+url="https://github.com/cpuguy83/go-md2man"
+arch="all"
+license="MIT"
+depends=""
+makedepends="go"
+#install=""
```

Comment:
useless line, please remove with empty `depends`

Reply:
latest commit should clean this up...  :)